internal/utils/test/test.go

```diff
@@ -1,64 +1,108 @@
 package testutils
 
 import (
-	"io/ioutil"
 	"os"
-	"strings"
 	"testing"
 
-	"github.com/mitchellh/go-homedir"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
 )
 
-// MustContainSubstring asserts the second provided arg is a substring of the first
-// TODO(REALMC-7315): make this a go-cmp Option instead
-func MustContainSubstring(t *testing.T, str, substr string) {
-	t.Helper()
-	if !strings.Contains(str, substr) {
-		t.Errorf("expected %q to be a substring of %q, but it was not", substr, str)
+// MustSkipf skips a test suite, but panics if REALM_NO_SKIP_TEST is set
+func MustSkipf(t *testing.T, format string, args ...interface{}) {
+	if len(os.Getenv("REALM_NO_SKIP_TEST")) > 0 {
+		panic("test was skipped, but REALM_NO_SKIP_TEST is set")
 	}
+	t.Skipf(format, args...)
 }
 
-// MustNotBeNil asserts the provided arg is not nil
-// TODO(REALMC-7315): make this a go-cmp Option instead
-func MustNotBeNil(t *testing.T, o interface{}) {
-	t.Helper()
-	if o == nil {
-		t.Error("expected value to not be <nil>, but it was")
-	}
+const (
+	defaultServerURL = "http://localhost:9090"
+)
+
+var realmServerRunning = false
+var realmServerNotRunning = false
+var skipUnlessRealmServerCalled = false
+var atlasServerRunning = false
+var atlasServerNotRunning = false
+var skipUnlessAtlasServerCalled = false
+
+// CloudAdminGroupID returns the Cloud admin group id to use for testing
+func CloudAdminGroupID() string {
+	return os.Getenv("REALM_MONGODB_CLOUD_GROUP_ID")
 }
 
-// MustMatch asserts the provided diff matches and fails the test if not
-func MustMatch(t *testing.T, o string) {
-	t.Helper()
-	if o != "" {
-		t.Error(o)
-	}
+// CloudAdminUsername returns the Cloud admin username to use for testing
+func CloudAdminUsername() string {
+	return os.Getenv("REALM_MONGODB_CLOUD_ADMIN_USERNAME")
```

Comment:
can we make this the same as baas? seems odd to have to define two different env vars for the same thing.

Reply:
So I carried over these env var names from the old `realm-cli` repo, primarily in an effort to reduce the friction of introducing this new code to the "same" evergreen build/environment.  Actually, to this point I had thought they were intentionally different to keep them separate.  Maybe @UnicodeSnowman could chime in here from a historical perspective?  I'd be happy to align them with `baas` if it doesn't break anything

Reply:
fyi: filed https://jira.mongodb.org/browse/REALMC-7449 to track the potential env var renaming work since this PR had some old/unrelated commits to it.  Resolving this thread here, for now